R/get-data.R

```diff
@@ -90,10 +98,66 @@ get_data_frame_glimpse <- function(x, x_name) {
 get_data_frame_print <- function(x) {
   withr::local_options(pillar.advice = FALSE, pillar.min_title_chars = Inf)
 
-  res <- cli::ansi_strip(capture.output(tibble::as_tibble(x, width = 1000, n = Inf)))
+  res <- cli::ansi_strip(
+    capture.output(tibble::as_tibble(x, width = 1000, n = Inf))
+  )
   res[2:length(res)]
 }
 
 get_data_frame_json <- function(x) {
   capture.output(jsonlite::toJSON(x, auto_unbox = TRUE, pretty = TRUE))
 }
+
+get_data_frame_skim <- function(df, .data_name = NULL) {
+  if (is.null(.data_name)) {
+    data_name <- deparse(substitute(df))
+  }
```

Comment:
```suggestion
```

It doesn't appear this is used later in the fn, maybe superseded by .data_name implemented in `get_data_frame()`?

Reply:
Ah thanks, I renamed the variable to match skimr's arg and missed this one